mica/archive/aca_l0.py

```diff
@@ -169,6 +170,63 @@ def get_slot_data(start, stop, slot, imgsize=None,
     return all_rows[oktime]
 
 
+def get_l0_images(start, stop, slot, imgsize=None, columns=None):
+    """
+    Get ACA L0 images for the given  ``start`` and ``stop`` times and
+    the given ``slot``.  Optionally filter on image size via ``imgsize``
+    or change the default image metadata via ``columns``.
+
+      >>> from mica.archive import aca_l0
+      >>> imgs = aca_l0.get_l0_images('2012:001', '2012:002', slot=7)
+      >>> imgs = aca_l0.get_l0_images('2005:001', '2005:002', slot=6, imgsize=[8])
+
+    :param start: start time of requested interval
+    :param stop: stop time of requested interval
+    :param slot: slot number integer (in the range 0 -> 7)
+    :param imgsize: list of integers of desired image sizes (default=[4, 6, 8])
+    :param columns: image meta-data columns
+
+    :returns: list of ACAImage objects
+    """
+    if columns is None:
+        columns = ['TIME', 'BGDAVG', 'IMGSTAT', 'IMGFUNC1', 'IMGSIZE']
+    if 'IMGROW0' not in columns:
+        columns.append('IMGROW0')
+    if 'IMGCOL0' not in columns:
+        columns.append('IMGCOL0')
+
+    slot_columns = list(set(columns + ['QUALITY', 'IMGRAW', 'IMGSIZE']))
+    dat = get_slot_data(start, stop, slot, imgsize=imgsize, columns=slot_columns)
+
+    ok = dat['QUALITY'] == 0
+    if not(np.all(ok)):
+        dat = dat[ok]
+
+    # Convert temperatures from K to degC
+    for temp in ('TEMPCCD', 'TEMPHOUS', 'TEMPPRIM', 'TEMPSEC'):
+        if temp in dat.dtype.names:
+            dat[temp] -= 273.15
+
+    # Masked array col access ~100 times slower than ndarray so convert
+    dat = dat.filled(-9999)
```

Comment:
`int` types don't support `NaN`.  

I've also been pondering a different approach which retains the original masked structured array and adds a new column for the `ACAImage` object.  Then one might pass a ref to the corresponding row into `ACAImage` to avoid the expense of converting the row into a dict.  This does create a reference cycle but probably OK.  This approach would eliminate the need for a new function.